handlers/bin/command-not-found

```diff
@@ -14,16 +19,16 @@ gettext.bindtextdomain('command-not-found', localedir='/usr/share/locale')
 gettext.textdomain(domain='command-not-found')
 
 
-def print_found(rows):
+def print_found(rows, term):
     print(file=sys.stderr)
     print(gettext.ngettext(
-        "The program '%(prog)s' can be found in the following package:",
-        "The program '%(prog)s' can be found in following packages:",
-        len(rows)) % ({'prog': term}), file=sys.stderr)
+        "The program '{prog}s' can be found in the following package:",
+        "The program '{prog}s' can be found in following packages:",
+        len(rows)).format(prog=term), file=sys.stderr)
 
     for row in rows:
-        print(_('  * %(prog)s [ path: %(path)s/%(binary)s, repository: %(repo)s ]') % \
-              ({'prog': row[1], 'path': row[2], 'binary': row[3], 'repo': row[0]}),
+        print(_('  * {prog}s [ path: {path}s/{binary}s, repository: {repo}s ]')
```

Comment:
I think that changing the strings will impact the translations, can you check if this is the case??

Reply:
Bah. That's a typo on my part. Good catch.